### PR TITLE
Pin a known to work pip version.

### PR DESCRIPTION
**Pull request by glogiotatidis**

### CODE REVIEW

**Comment by glogiotatidis** [comment]
@jgmize r?

File: Dockerfile [hunk 1]
```diff
@@ -1,12 +1,19 @@
-FROM python:3.4-slim
+FROM python:3.3-slim
```
Comment (jgmize) [inline Dockerfile]: Why the python downgrade?

Comment (glogiotatidis) [inline Dockerfile]: https://github.com/glogiotatidis/masterfirefoxos/commit/c4d53ece5daf5939fe4aed9632a389aba7036074

Comment (jgmize) [inline Dockerfile]: Ah, thanks.

**Pull request merged**